index.bs

```diff
@@ -5516,7 +5516,7 @@ Each stored [=virtual authenticator=] has the following properties:
         Note: This string can be a UUID [[RFC4122]].
     </div>
 : |protocol|
-:: The protocol the [=Virtual Authenticator=] uses to encode its messages. Either `ctap2` for the [[FIDO-CTAP]] format
+:: The protocol the [=Virtual Authenticator=] speaks: either `ctap2` or `ctap1-u2f` [[FIDO-CTAP]].
```

Comment:
we probably should s/ `ctap1-u2f` / `ctap1/u2f` /

Reply:
Done!